brave-jaxrs2/README.md

```diff
@@ -12,18 +12,36 @@ the request header. Also sends sr annotations.
 Also sends cs annotations.
 *   `BraveClientResponseFilter` - Intercepts JAX-RS 2.x client responses and sends cr annotations. Also submits the completed span.
 
+For convenience there is also a JAX-RS Feature BraveTraceFeature that can be added to clients as well as services.
+
 ## Usage ##
 
 For server side setup, you simply need to tell JAX-RS 2.x to scan the com.github.kristofa.brave.jaxrs2 package, and
 the container filters will be picked up and registered automatically.
 
-for Resteasy 3 and Jersey 2 examples check corresponding modules documentation
+for Resteasy 3 and Jersey 2 examples check corresponding modules documentation.
+
+If you can not use auto discovery then you can add the feature to the service manually.
+
+For example in an Application class this looks like:
+
+	public class MyApplication extends Application {
 
-For client side setup, you just have to register the client filters with your JAX-RS 2 client before you make your request.
+	    public Set<Object> getSingletons() {
```

Comment:
is this an override? if so add `@Override` just so it is clear (just don't know if this is a magic return or a part of the super)